DECLARING_DEPENDENCIES.md

```diff
@@ -0,0 +1,210 @@
+# Declaring Dependencies on Google Java Libraries
+
+Google maintains a number of open source Java libraries that make it
+easier to use services in Google Cloud Platform (GCP). Additionally Google
+maintains several foundational libraries that can be used for very
+general purposes, which the GCP libraries also depend on. A user of
+these libraries needs to use compatible versions of them in order to
+avoid dependency conflicts. This document explains how to use BOMs
+(bill of materials) to accomplish that and explains additional
+practices that make dependency management go more smoothly.
+
+These recommendations apply to the usage of the following libraries:
+
+- [guava](https://github.com/google/guava)
+- [protobuf](https://github.com/protocolbuffers/protobuf)
+- [grpc-java](https://github.com/grpc/grpc-java)
+- [gax-java](https://github.com/googleapis/gax-java)
+- [google-http-java-client](https://github.com/googleapis/google-http-java-client)
+- [google-oauth-java-client](https://github.com/googleapis/google-oauth-java-client)
+- [google-api-java-client](https://github.com/googleapis/google-api-java-client)
+- [google-cloud-java](https://github.com/googleapis/google-cloud-java)
+- [beam](https://github.com/apache/beam)
+
+## Ensuring Compatibility
+
+If you depend on one or more of these libraries, in order to
```

Comment:
"If you depend on one or more of these libraries, in order to
ensure that your own project uses compatible versions of them," --> "To
ensure that your own project uses compatible versions of these libraries,"